_posts/2017-07-24-combining-free-monads-in-haskell.html

```diff
@@ -314,4 +314,4 @@
 	<p>
 		<strong>Next:</strong> Combining free monads in F#.
 	</p>
-</div>
\ No newline at end of file
+</div>
```

Comment:
Hmm... GitHub's built-in editor shows this as changed, even though I've only edited line 138. Weird...